src/Common/src/System/Net/ContextAwareResult.netcore50.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Thought about that.  My imppression was the networking team would want to fill in the real implementation for unix.